style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 50px;
+  font-family: "Nata Sans", sans-serif;
+}
+
+
+body {
+  background-image: url(back-img.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  justify-content: center; 
+  align-items: center; 
+  font-size: 25px;  
+}
+
+
+.theWelcome {
+  background-image: url(main-img.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 50%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;  
+  align-items: center;    
+  text-align: center;  
+  color: white;
+}
+.theWelcome p {
+  margin-top: 2em;
+  font-size: 35px;
+}
+
+.theWelcome p:nth-of-type(2) {
+  font-size: 12px;
+  text-align: center;
+  max-width: 63%;
+  margin-top: 1em;
+}
+
+.mainBox {
+  width: 30%;
+  height: 70%;
+  border-radius: 1em;
```

Comment:
in general, i recomend you to use one of two ways for size in css - percentage (%) if you want it to be responsive to the page size; and pixels (px) if you want it to fixed sized